.github/workflows/run-archiver.yml

```diff
@@ -78,6 +73,7 @@ jobs:
           path: ${{ matrix.dataset }}_run_summary.json
 
   archive-run-large:
+    if: inputs.large_runner
```

Comment:
If set as true in workflow dispatch, or triggered by scheduled run this _should_ run.

Reply:
My assumption was that an empty string would actually get evaluated as true, but I could be totally off-base here. I think your suggestion re: incorporating the type of run below is great and I'll incorporate it here.

Reply:
I *think* an unset variable here would get treated as a "falsey" value: https://docs.github.com/en/actions/learn-github-actions/expressions#literals 

> Note that in conditionals, falsy values (false, 0, -0, "", '', null) are coerced to false and truthy (true and other non-falsy values) are coerced to true.

And actually I bet unset variable is actually `null` instead of `''`, now that I look at those docs.

Reply:
Either way, making this more explicit seems wise.